PCbuild/readme.txt

```diff
@@ -252,12 +252,13 @@ against a profiling library and contain extra debug information. The
 PGUpdate configuration takes the profiling data and generates optimized
 binaries.
 
-The build_pgo.bat script automates the creation of optimized binaries.
-It creates the PGI files, runs the unit test suite or PyBench with the
-PGI python, and finally creates the optimized files.
+The build.bat script has arguments `--pgo` and `--pgo-job <job>` that 
+automate the creation of optimized binaries. It creates the PGI files, 
+runs the unit test suite or PyBench with the PGI python, and finally 
+creates the optimized files. Try `build.bat -h` to learn more.
```

Comment:
This file is where people come to learn even more than in `build.bat -h`, so we should include more info here rather than point people to that command.